internal/ask/form/form_test.go

```diff
@@ -0,0 +1,115 @@
+package form_test
+
+import (
+	"log"
+	"os"
+	"testing"
+
+	"github.com/ardanlabs/kit/cfg"
+	"github.com/ardanlabs/kit/db"
+	"github.com/ardanlabs/kit/db/mongo"
+	"github.com/ardanlabs/kit/tests"
+	"github.com/coralproject/shelf/internal/ask/form"
+	"github.com/coralproject/shelf/internal/ask/form/formfix"
+)
+
+var dbSession *db.DB
+
+// prefix is what we are looking to delete after the test.
+const prefix = "FTEST_"
+
+func TestMain(m *testing.M) {
+	// Initialize the configuration and logging systems. Plus anything
+	// else the web app layer needs.
+	tests.Init("XENIA")
```

Comment:
The scoping to `FTEST` allows this package's tests to isolate it's own documents that it creates. Ideally as well, we shouldn't need to depend on removing/cleaning up after the test as that should be the responsibility of the CI server. So we'd likely remove the prefix all together at that point.